compiler/frontend/beam/src/main/java/org/apache/nemo/compiler/frontend/beam/PipelineTranslator.java

```diff
@@ -169,7 +240,7 @@ private static void windowTranslator(final TranslationContext ctx,
     } else {
       throw new UnsupportedOperationException(String.format("%s is not supported", transform));
     }
-    final IRVertex vertex = new OperatorVertex(new WindowTransform(windowFn));
+    final IRVertex vertex = new OperatorVertex(new WindowDoFnTransform(windowFn));
```

Comment:
Can we use the existing name `WindowTransform`? 
Beam's `WindowFn` doesn't seem to extend `DoFn`.

Reply:
How about `WindowFnTransform`?